docs/components/connectors/out-of-the-box-connectors/automation-anywhere.md

```diff
@@ -71,7 +71,7 @@ The **Automation Anywhere Connector** currently supports two operation types in
 ### Add work item to the queue
 
 This operation provides the ability to add a work queue item in the specified queue.
-It corresponds directly to the respective Automation Anywhere API - [`Add Work Items to the queue API`](https://docs.automationanywhere.com/bundle/enterprise-v2019/page/enterprise-cloud/topics/control-room/control-room-api/cloud-api-wlm-add-workitems.html).
+It corresponds directly to the respective Automation Anywhere API - [`Add Work Items to the queue API`](https://docs.automationanywhere.com/bundle/enterprise-v11.3/page/enterprise/topics/control-room/control-room-api/add-work-item-data-to-queue-api.html).
```

Comment:
This is the latest supported documentation, and that's how I went about adjusting all the URLs. If I'm updating on a monthly basis, I should be repeatedly checking links like these so don't mind maintaining if the latest version changes or another version is no longer supported.

If you have better ideas, definitely let me know!

Reply:
Unless it's a dependency where the version would tied to a specific version of C8, the current or latest docs make the most sense. 

My dream would be to get all these third party links in a single file and use variables across the docs content instead, so we only need to update them in one place, but if you'll bake this into monthly work, that will do for now!

Reply:
I agree here. I'm meeting with Steve today anyways, so I'll ask him about how I can get started on a file that fills these variables in and adjust in this PR alongside the comments from @RomanJRW.

Reply:
After chatting with Steve, we're going to hold off on a file outlining these links (and subbing in variables) for now, for one largely to wait and see if there is a large need for one after a few rounds of these third-party link reviews in the months ahead if most link references are just made in 3-4 places rather than, say, 40 or 50. For another, we're worried the file could get messy, particularly if new documentation doesn't follow these link references and variable injections and we have scattered usage.